internal/pkg/store/ssm/ssm.go

```diff
@@ -66,6 +69,9 @@ func NewStore() (*SSM, error) {
 	return &SSM{
 		systemManager: ssm.New(sess),
 		identity:      identity.New(sess),
+		// If the region is not set to us-east-1, we can't find out if a domain name exists in the account.
```

Comment:
wat =P

Reply:
I think what we're trying to say here is Route 53 Domains is only available in `us-east-1`?